src/components/light/light.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+import off from "./light-off.svg";
+import on from "./light-on.svg";
+import "./style.css";
+
+const image = [off, on];
+
+const Light = ({ name, state }) => {
+  const [stav, setStav] = useState({ state });
```

Comment:
máš všechna světla ze začátku zapnutá a měla by být zapnutá jen 2 (viz smartHomeData.js). čím to asi bude? hint: zkus si vypsat (console.log) co je v proměnných `state` a `stav`